### DISCUSSION TITLE
Can I show the successful tests in output?

### DISCUSSION THREAD

**Discussion by l3112** [aquasecurity/trivy#4598] [Q&A]
### Question

When I run a scan, I see:

`Tests: 10 (SUCCESSES: 1, FAILURES: 9, EXCEPTIONS: 0)
Failures: 9 (UNKNOWN: 0, LOW: 1, MEDIUM: 2, HIGH: 6, CRITICAL: 0)`

And the output shows all of the failures.

Is there a flag I can use in my command to see the success to better understand what's working in my custom rego policy? 

### Target

AWS

### Scanner

Misconfiguration

### Output Format

None

### Mode

Standalone

### Operating System

macOS BigSur

### Version

```bash
0.41.0
```

**Comment by itaysk** [comment]
duplicate of https://github.com/aquasecurity/trivy/issues/4426
will likely get added soon https://github.com/aquasecurity/trivy/pull/4427
